Editor/Resources/KeyBindings/CreatorDefaultKeyBindings.inputactions

```diff
@@ -0,0 +1,115 @@
+{
+    "name": "CreatorDefaultKeyBindings",
+    "maps": [
+        {
+            "name": "Default",
+            "id": "4907d911-b591-428c-8409-4b57ad6b6293",
+            "actions": [
+                {
+                    "name": "MoveToNextCamera",
```

Comment:
I think this is one of the only things that bother me. Changes or extension on the input in advanced or basic ui will cause changes in the Core :/ thinking about a better way